src/tools/compiletest/src/common.rs

```diff
@@ -384,6 +384,9 @@ pub struct Config {
     pub only_modified: bool,
 
     pub target_cfg: LazyCell<TargetCfg>,
+
+    /// Nocapture flag passed
```

Comment:
This looks more like a commitment information rather than a doc-comment.

Reply:
update: I meant the message itself doesn't look so right. We don't usually document things like "passed, sent, etc". You can even remove the comment since the field name explain itself good enough.

Reply:
Ah, I misunderstood then.